x/stake/keeper/key.go

```diff
@@ -71,8 +71,15 @@ func GetBondedValidatorIndexKey(operator sdk.ValAddress) []byte {
 func getValidatorPowerRank(validator types.Validator) []byte {
 
 	potentialPower := validator.Tokens
-	powerBytes := []byte(potentialPower.ToLeftPadded(maxDigitsForAccount)) // power big-endian (more powerful validators first)
```

Comment:
can to delete `ToLeftPadded` now I believe

Reply:
Hmm it could be useful in general for other serialization purposes though.